flask_rebar/rebar.py

```diff
@@ -690,7 +690,7 @@ def create_handler_registry(
             the Swagger generator that is used in the endpoints swagger and swagger UI
             that are added to the API.
             If left as None, a `SwaggerV2Generator` instance will be used.
-        :param str swagger_path:
+        :param str spec_path:
             The Swagger specification as a JSON document will be hosted at this URL.
```

Comment:
Update this pydoc as well. Instead of "Swagger specification" would be "Open API specification"